plugins/node/opentelemetry-instrumentation-mongodb/src/types.ts

```diff
@@ -15,6 +15,11 @@
  */
 
 import { InstrumentationConfig } from '@opentelemetry/instrumentation';
+import { Span } from '@opentelemetry/api';
+
+export interface MongoDBInstrumentationExecutionResponseHook {
+  (span: Span, result: CommandResult): void;
```

Comment:
The current hook signature is only sending the result object from the driver, but future usage may require adding additional data to the hook (like instrumented module version).
To avoid having a function with a long signature and many parameters (which user only cares about few of them), it was suggested to have a `responseInfo` parameter with the relevant data in it.

[Link to discussion](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/415#issuecomment-824277696)
[Example](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/plugins/node/opentelemetry-instrumentation-ioredis/src/types.ts#L49)

Reply:
I agree with you, added `MongoResponseHookInformation` interface